reference/orders.v2.oas2.yml

```diff
@@ -182,6 +202,35 @@ paths:
                         value: 12
                       price_inc_tax: 12.45
                       price_ex_tax: 10.12
+              Updating an Order's Shipping Consignment:
+                value:
+                  consignments:
+                     billing_address:
+                        first_name: "John"
+                        last_name: "Doe"
+                        company: "BC"
+                        street_1: "12 Foo St"
+                        city: "Austin"
+                        state: "Texas"
+                        zip: "78751"
+                        country: "United States"
+                        country_iso2: "US"
+                        email: "test@tny.mx"
+                     shipping:
+                        id: 149
+                        line_items:
+                          - id: 123
+                            quantity: 1
+                        first_name: "John"
+                        last_name: "Doe"
+                        company: "BC"
+                        street_1: "12 Foo St"
+                        city: "Austin"
+                        state: "Texas"
+                        zip: "78751"
+                        country: "United States"
+                        country_iso": "US"
+                        email: "john.doe@bigcommerce.com"
```

Comment:
there is no "billing_address" in consignments
also the "shipping" should be an array

Reply:
Thanks for the call out, fixed this ✅